Makefile

```diff
@@ -0,0 +1,39 @@
+########################################################
+
+NAME := ftl_installer
+
+# VERSION file provides one place to update the software version.
+VERSION := $(shell cat VERSION)
+
+__init__.py: ftl_installer/__init__.py.in
+	sed "s/%VERSION%/$(VERSION)/" $< > ftl_installer/__init__.py
+
+setup.py: setup.py.in
+	sed "s/%VERSION%/$(VERSION)/" $< > setup.py
+
+venv: clean __init__.py setup.py
+	@echo "#############################################"
+	@echo "# Creating a virtualenv"
+	@echo "#############################################"
+	virtualenv venv
```

Comment:
I think it would have to be, yes.
On Mar 15, 2016 9:54 AM, "Andrew Butcher" <notifications@github.com> wrote:

> In Makefile
> <https://github.com/ftl-toolbox/ftl_installer/pull/3#discussion_r56166313>
> :
>
> > +NAME := ftl_installer
> > +
> > +# VERSION file provides one place to update the software version.
> > +VERSION := $(shell cat VERSION)
> > +
> > +__init__.py: ftl_installer/__init__.py.in
> > +	sed "s/%VERSION%/$(VERSION)/" $< > ftl_installer/__init__.py
> > +
> > +setup.py: setup.py.in
> > +	sed "s/%VERSION%/$(VERSION)/" $< > setup.py
> > +
> > +venv: clean __init__.py setup.py
> > +	@echo "#############################################"
> > +	@echo "# Creating a virtualenv"
> > +	@echo "#############################################"
> > +	virtualenv venv
>
> How do you see this being specified - environment variable?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/ftl-toolbox/ftl_installer/pull/3/files#r56166313
>